crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -748,6 +748,20 @@ en:
       non_json_file: 'Unprocessable chef file found'
       invalid_file_extension: 'Filename needs to be a valid tarball with a .tar.gz file extension'
 
+  sanities:
+    show:
+      header: 'Sanity Check Failures'
+      notice: |
+        There are errors that must be resolved. Please restart Crowbar after solving the errors.
```

Comment:
I think it depends on what checks we use, and which one fails. It is a bit strange from UX I agree, so maybe we can drop this suggestion for now until we have a check where a restart is really necessary, what do you say?